<launch>
    <node pkg="orb_feature_controller" type="orb_feature_controller_node" name="orb_feature_controller_node" output="screen">
        <!--Camera config-->
        <remap from="/camera/color/image_raw" to="/camera/color/image_raw"/>
        <remap from="/camera/color/image_raw" to="/camera/color/image_raw"/>
        <remap from="/camera/aligned_depth_to_color/image_raw" to="/camera/aligned_depth_to_color/image_raw" />

        <!--Cmd Velocity config-->
        <remap from="/turtle1/cmd_vel" to="/cmd_vel" />

        <param name="/max_linear_velocity" type="double" value="0.22"/>
        <param name="/max_angular_velocity" type="double" value="2.84"/>
        <param name="/linear_step_size" type="double" value="0.01"/>
        <param name="/angular_step_size" type="double" value="0.1"/>

        <!--debug output config-->
        <remap from="/overlay_image" to="/overlay_image" />

        <!--Feature controller config-->
        <param name="/RoI" type="double" value="0.5"/>
        <param name="/nFeatures" type="int" value="2000"/>
        <param name="/distance_limit" type="int" value="20"/>
        <param name="/error_limit" type="double" value="10.0"/>
        <param name="/good_followed_num" type="int" value="11"/>
    </node>
    <node pkg="hector_trajectory_server" type="hector_trajectory_server" name="trajectory_server" output="screen">
         <param name="/target_frame_name" value="odom" />
         <param name="/source_frame_name" value="base_footprint" />
         <param name="/trajectory_update_rate" value="20.0" />
         <param name="/trajectory_publish_rate" value="20.0" />
    </node>
</launch>